optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingCfpDevoxxImporter.java

```diff
@@ -221,128 +219,36 @@ private void importSpeakerList() {
                         + ") already exists in the speaker list");
             }
             speakerNameToSpeakerMap.put(speakerName, speaker);
-
-            JsonArray speakerTalksArray = speakerObject.getJsonArray("acceptedTalks");
-            for (int j = 0; j < speakerTalksArray.size(); j++) {
-                String talkUrl = speakerTalksArray.getJsonObject(j).getJsonArray("links").getJsonObject(0).getString("href");
-                talkUrlSet.add(talkUrl);
-            }
         }
 
         speakerList.sort(Comparator.comparing(Speaker::getName));
         solution.setSpeakerList(speakerList);
     }
 
-/* TODO: Uncomment this when the REST api exposes the talks
     private void importTalkList() {
         this.talkCodeToTalkMap = new HashMap<>();
-        List<Talk> talkList = new ArrayList<>();
-        Long talkId = 0L;
+        solution.setTalkList(new ArrayList<>());
+
+        String talksUrl = conferenceBaseUrl + "/talks";
+        LOGGER.debug("Sending a request to: " + talksUrl);
```

Comment:
Use logger.debug("Sending a request to: {}", talksUrl);
The difference is that the string concatenation won't happen if the logger isn't set to debug, saving performance. Not that you'll notice the perf difference, but it's a good practice to do it right everywhere.